tests/unit/StoreCurrentAccount.spec.ts

```diff
@@ -1,47 +1,32 @@
-import { getVuexStore } from '@/store'
+import { createPinia, setActivePinia } from 'pinia'
+import { useAuthenticationStore } from '@/stores/authenticationStore'
 import { shallowMount, createLocalVue } from '@vue/test-utils'
-import Vue, { nextTick } from 'vue'
+import Vue from 'vue'
 import Vuetify from 'vuetify'
 
 Vue.use(Vuetify)
 const localVue = createLocalVue()
+setActivePinia(createPinia())
+const authenticationStore = useAuthenticationStore()
 
 describe('testing current account module', () => {
-  const dummyAccount = (accountType: string) => {
-    return {
-      accountStatus: 'string',
-      accountType: accountType,
-      additionalLabel: 'string',
-      id: 1234,
-      label: 'string',
-      productSettings: 'string',
-      type: 'string',
-      urlorigin: 'string',
-      urlpath: 'string'
-    }
-  }
-
   it('fetches current account from session storage - PREMIUM', async () => {
     jest.isolateModules(() => async () => {
       const SbcHeader = require('sbc-common-components/src/components/SbcHeader.vue').default
-      const store = await getVuexStore() as any // remove typings for unit tests
-      shallowMount(SbcHeader, { store, localVue })
+      shallowMount(SbcHeader, { localVue })
 
-      store.commit('account/setCurrentAccount', dummyAccount('PREMIUM'), { root: true })
-      expect(store.getters.isPremiumAccount).toBe(true)
-      expect(store.getters.isSbcStaff).toBe(false)
+      expect(authenticationStore.isPremiumAccount).toBe(true)
+      expect(authenticationStore.isSbcStaff).toBe(false)
```

Comment:
I don't understand how this is working, since you don't set the account type ...?

Reply:
I'm not really sure tbh but the tests are passing 🤔

Reply:
I played a bit with this and this test isn't working at all -- the getters aren't being called.

I looked at where else the authentication store is used: only the Add Staff Notation dialog -- and it's broken (staff filings dropdown doesn't display because `getKeycloakGuid` returns undefined.

This needs to be fixed in this PR (we can't leave it broken for any time in Dev).

Reply:
So, I've spent my entire afternoon trying to fix this as I made this my top priority since it's super important.

The problem wasn't with the `getKeycloakGuid` as I fixed it but it was still broken. 

The way I found the problem was by cloning my repo in a different folder and checking out commit by commit. I was sure that the staff filings dropdown was working at a certain point and one of the commits actually broke it. Each time I checked out a commit, I ran the project and checked if the dropdown was broken.

The problem was in the 'Update part 2 - lots of code cleanup" commit. Specifically, in the `Signin.vue` file.
https://github.com/bcgov/business-filings-ui/commit/a217fa1240a19ab87db6a8c97bff174e5fd9f4d1#diff-e1ac6403d23a844d2b27c3d914115fcdd0a72be080e4ddf9c2d699c6e9e37793

I removed the `@Component` since I thought we don't need it since there's no computed property anymore. It turns out, we need it although there isn't anything in it. 

Sorry for the long comment and explanation. Just felt like I should share my findings :)

Reply:
Nice find. Good job finding it. Sorry I didn't notice it either.